### ISSUE TITLE
Getting DeprecationWarning in webpack 2

### ISSUE THREAD

**Issue by jacobbuck** [deepsweet/mustache-loader#21]
```
(node:67349) DeprecationWarning: loaderUtils.parseQuery() received a non-string value which can be problematic, see https://github.com/webpack/loader-utils/issues/56
parseQuery() will be replaced with getOptions() in the next major version of loader-utils.
```

In my webpack.config.js:

```js
{
  test: /\.mustache$/,
  exclude: ['node_modules'],
  use: [{
    loader: 'mustache-loader',
    options: { noShortcut: true }
  }]
},
```

**Comment by baderbuddy** [comment]
Thank you very much, I think I have some code in place to fix it, I just need to test it.

**Comment by baderbuddy** [comment]
This has been fixed in version 1.0.0
